### DISCUSSION TITLE
Running without a desktop manager seems to run on wrong display

### DISCUSSION THREAD

**Discussion by ian-uson** [rustdesk/rustdesk#5736] [General]
### Bug Description

When running a pure openbox session with /etc/X11/Xsession , with no desktop manager, rustdesk will create a new x server instead of displaying what is on the main display (:0).

### How to Reproduce

Use a server installation of debian without a desktop manager, 

run an x session using /etc/X11/Xsession with an openbox-session on :0, then run rustdesk as normal from the console

### Expected Behavior

should run normally, using the display that is active

### Operating system(s) on local side and remote side

Ubuntu -> Debian

### RustDesk Version(s) on local side and remote side

1.2.2->1.2.2

### Screenshots

![Screenshot from 2023-09-18 08-31-19](https://github.com/rustdesk/rustdesk/assets/142534879/d183ade6-957c-4299-8ec0-c6627dfa4763)

**Comment by ian-uson** [comment]
Also gives error "Unsupported display server" error, even though x is running. is there an environnment variable i can set to bypass this and force it to know what my x server is? I'm looking through the source code, and have tried setting XDG_SESSION_ID, XDG_SESSION_TYPE, XDG_RUNTIME_DIR manually without any success.

![image](https://github.com/rustdesk/rustdesk/assets/142534879/6e71b32e-bfda-474c-ba75-506ca7acf206)

**Comment by ian-uson** [comment]
okay, got it working by suppyling XDG_SESSION_TYPE to the service as well as the user executable

**Reply by rustdesk** [reply]
what `XDG_SESSION_TYPE` do you set?

**Reply by ian-uson** [reply]
XDG_SESSION_TYPE=x11

Though i've run into another weird issue. it seems to do this same thing until i log in (on an entirely different console unrelated to rustdesk). i'm very confused. It has the banner saying i need to use x11 on the left side forever (until i open a console and log in, then the banner immediately goes away and it starts working). 
Any ideas on what logging in is doing that i can do manually to get it to work on startup?

**Reply by ian-uson** [reply]
seems to be related to this maybe. https://github.com/rustdesk/rustdesk/issues/73

before i log in inside the consol, i get no sessions from loginctl. once i log in, it gives me a session. is this actually necessary though?
